src/extension/extensionHost.ts

```diff
@@ -30,24 +30,37 @@ const consoleLogger: Logger = {
     },
 }
 
+/**
+ * Required information when initializing an extension host.
+ */
+export interface InitData {
+    /** The URL to the JavaScript source file (that exports an `activate` function) for the extension. */
+    bundleURL: string
+
+    /** @see {@link module:sourcegraph.internal.sourcegraphURL} */
+    sourcegraphURL: string
```

Comment:
As it turns out, this sourcegraphURL is not even needed to do what I intended to do. I will remove it.

@chrismwendt Did you see https://github.com/sourcegraph/sourcegraph-langserver-http/pull/5/files#diff-70d8241068a7f9eda07d1753ee7e181fR40? I don't think you need a fallback URL because the extension won't `fetch()` directly, it'll call the `queryLSP` command (which the client registers and implements).

Reply:
Oh cool, so that fallback logic will live in browser-extensions? I think https://github.com/sourcegraph/browser-extensions/pull/162/files#diff-2862b04682a2d6c1afbb49534ea8f693R47 will need to fall back (I can make that change once that PR is merged in).

Reply:
Yeah. I actually didn't think of the fallback logic here. I will call that out as a separate TODO (soon) and probably not address that in this PR. It will only affect users who opt into "Use Sourcegraph extensions" and it is broken now for those users anyway.